packages/office-ui-fabric-react/src/components/ActivityItem/ActivityItem.tsx

```diff
@@ -3,12 +3,14 @@ import { IActivityItemProps } from './ActivityItem.types';
 import { IActivityItemClassNames, getClassNames } from './ActivityItem.classNames';
 import { getStyles } from './ActivityItem.styles';
 import { PersonaSize, PersonaCoin, IPersonaSharedProps, IPersonaCoinProps } from '../../Persona';
+import { customizable } from '../../Utilities';
 
 type OptionalReactKey = { key?: React.Key };
 
 /**
  * {@docCategory ActivityItem}
  */
+@customizable('ActivityItem', ['theme', 'styles'], true)
```

Comment:
this is why the bundle size increased... it should be fine but perhaps we can consider alternatives?

Reply:
Oh I see, hmm do you have any ideas on alternatives for using this decorator?

Reply:
Hey @aneeshack4! do you have any suggestions on alternatives?